crypto/cmp/cmp_server.c

```diff
@@ -57,6 +58,7 @@ OSSL_CMP_SRV_CTX *OSSL_CMP_SRV_CTX_new(OSSL_LIB_CTX *libctx, const char *propq)
 
     if ((ctx->ctx = OSSL_CMP_CTX_new(libctx, propq)) == NULL)
         goto err;
+    ctx->certReqId = OSSL_CMP_CERTREQID_NONE - 1;
```

Comment:
Why `OSSL_CMP_CERTREQID_NONE - 1`? Is it so an increment gets us to `OSSL_CMP_CERTREQID_NONE`? (I couldn't see that done, but might be missing something) or is this basically `OSSL_CMP_CERTREQID_INVALID`

Reply:
So is there a reason not to create an `OSSL_CMP_CERTREQID_INVALID` macro? If only to avoid someone using the value -2 in the future...

Reply:
Good point - done.